midealocal/devices/a1/__init__.py

```diff
@@ -31,16 +31,22 @@ class DeviceAttributes(StrEnum):
 
 
 class MideaA1Device(MideaDevice):
-    _modes = ["Manual", "Continuous", "Auto", "Clothes-Dry", "Shoes-Dry"]
-    _speeds = {
+    _modes: ClassVar[list[str]] = [

```

Comment:
```suggestion
    _modes: list[str] = [
```
Do this work?

Reply:
Nope unfortunately, this is the ref. <https://docs.astral.sh/ruff/rules/mutable-class-default/>